pkg/cli/setup/application.go

```diff
@@ -20,16 +20,19 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+
+	"github.com/radius-project/radius/pkg/version"
 )
 
 const (
-	appBicepTemplate = `import radius as radius
+	appBicepTemplate = `provider radius
```

Comment:
FYI it looks like this will change again to `extension radius`.

Reply:
Got it, will update as part of this PR or in a follow up PR depending on when they release the next version with that change